packages/manager/modules/pci/src/components/project/flavor-billing/flavor-billing.controller.js

```diff
@@ -1,16 +1,32 @@
 export default class FlavorBillingController {
+  /* @ngInject */
+  constructor(TranslateService) {
+    this.TranslateService = TranslateService;
+  }
+
   $onInit() {
     this.monthly = this.monthlyBilling === true;
     this.number = this.number || 1;
 
     this.disabled = this.disabled || false;
   }
 
+  set flavor(flavor) {
+    if (flavor) {
+      this.prices = flavor.prices;
+      this.PriceFormatter = new Intl.NumberFormat(this.TranslateService.getUserLocale().replace('_', '-'), { style: 'currency', currency: flavor.prices.hourly.currencyCode });
```

Comment:
Don't you still format using locale ? 🤔 

Also, are you sure it works for each locale ? Because the OVH api doesn't respect `BCP 47` :(

Reply:
The locale is get from our LANGUAGES constants, however I cannot guarantee the format of the `currencyCode`, I'll have a look into it